pkg/pipelinerunmetrics/metrics.go

```diff
@@ -272,9 +294,21 @@ func (r *Recorder) RunningPipelineRuns(lister listers.PipelineRunLister) error {
 	}
 
 	var runningPRs int
+	var trsWaitResolvingTaskRef int
+	var prsWaitResolvingPipelineRef int
+
 	for _, pr := range prs {
 		if !pr.IsDone() {
 			runningPRs++
+			succeedCondition := pr.Status.GetCondition(apis.ConditionSucceeded)
+			if succeedCondition != nil && succeedCondition.Status == corev1.ConditionUnknown {
+				switch succeedCondition.Reason {
+				case v1.TaskRunReasonResolvingTaskRef:
+					trsWaitResolvingTaskRef++
+				case "ResolvingPipelineRef": // unfortunately importing pipelinerun.ReasonResolvingPipelineRef results in a circular dependency
```

Comment:
Only because I was being conservative and attempting to minimize the delta of the change, as I did not research why it was not put there to begin with.

As I inferring yours is, my guess is it was just oversight wrt not putting it in v1, but I'll try to do some git file history investigation, and just make sure there was no weird rationale for it, and assuming there is no unexpected reason why, I'll move it.

Thanks.

Reply:
sounds good; I don't know of any specific reason for not putting it in the v1 package but looking through the commit history would definitely be helpful.

Reply:
Yes, I feel this can be a separate issue.

Reply:
I figured it was easier to just do this: https://github.com/tektoncd/pipeline/pull/7102
I'd prefer the cleanup PR be merged first but either way is OK.

Reply:
Assuming #7102 will get merged in the near team, I'm good with holding this PR until that happens, and then rebasing this and getting rid of the literal string usage @lbernick 

Thanks